.github/workflows/build.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           npm run lint
           npm run test:unit
       - name: Code Coverage Report
-        uses: SonarSource/sonarcloud-github-action@master
+        uses: SonarSource/sonarcloud-github-action@9f9bba2c7aaf7a55eac26abbac906c3021d211b2 # master
```

Comment:
don't user master, otherwise:
* can fail if the dev team makes a mistakes and master doesn't run
* dependabot will pr each time there is a push in master.

It's recommended to use the latest release which in this case is 3.1.0